src/Apps/Artist/Routes/Overview/state.ts

```diff
@@ -20,11 +20,21 @@ export class FilterState extends Container<State> {
   }
 
   setMajorPeriods(value) {
-    this.setState({ major_periods: [value], page: 1 })
+    this.setState({
```

Comment:
Basically, using the state file here we control unsetting the other filters when selecting one, etc.